app/controllers/reservations_controller.rb

```diff
@@ -192,18 +192,18 @@ def edit
   def update # rubocop:disable all
     message = 'Successfully edited reservation.'
     res = reservation_params
-    # add new equipment object id to hash if it's being changed and save old
+    # add new equipment item id to hash if it's being changed and save old
     # and new objects for later
-    unless params[:equipment_object].blank?
-      res[:equipment_object_id] = params[:equipment_object]
-      new_object = EquipmentObject.find(params[:equipment_object])
+    unless params[:equipment_item].blank?
+      res[:equipment_item_id] = params[:equipment_item]
+      new_object = EquipmentItem.find(params[:equipment_item])
```

Comment:
you should probably rename the variable names to `new_item` and `old_item`, etc.